.github/workflows/publish-server.yml

```diff
@@ -5,6 +5,8 @@ on:
     branches: ['main']
     tags:
       - libsql-server-v*.*.*
+  # REMOVE BEFORE MERGING
+  pull_request:
```

Comment:
@QuiiBz I think now is the time to remove this 🤣

Reply:
Indeed, looks like the job passed!

<img width="743" alt="Screenshot 2024-01-02 at 16 24 04" src="https://github.com/tursodatabase/libsql/assets/43268759/6d8afb5e-0dc4-487d-9975-edf275ebe24a">